src/main/java/ui/components/KeyboardShortcuts.java

```diff
@@ -1,61 +1,49 @@
 package ui.components;
 
-import javafx.application.Platform;
 import javafx.scene.input.KeyCode;
 import javafx.scene.input.KeyCodeCombination;
 import javafx.scene.input.KeyCombination;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import prefs.Preferences;
-import util.DialogMessage;
 
 import java.util.HashMap;
 import java.util.Collections;
-import java.util.HashSet;
 import java.util.Map;
-import java.util.Set;
 
 /**
- * a central place to specify keyboard shortcuts
- * <p>
- * Classes that currently have keyboard shortcut code:
- * ui.components.NavigableListView
- * ui.issuepanel.PanelControl
- * ui.listpanel.ListPanel
- * ui.MenuControl
- * <p>
- * Utility Class:
- * util.KeyPress
+ * This class handles the keyboard shortcuts used by the different UI components
```

Comment:
I feel like that's clear from everything in the class being static and final, but I guess it's subjective.